diff --git a/lib/_tsc.js b/lib/_tsc.js
index 11ab5ff444ede75f643b3df775438d35d56fb9bd..430031522df8728d0b0a1179b24de969681999b1 100644
--- a/lib/_tsc.js
+++ b/lib/_tsc.js
@@ -80238,16 +80238,6 @@ function createTypeChecker(host) {
         [effectiveLeft, effectiveRight] = getBaseTypesIfUnrelated(leftType, rightType, isRelated);
       }
       const [leftStr, rightStr] = getTypeNamesForErrorDisplay(effectiveLeft, effectiveRight);
-      if (!tryGiveBetterPrimaryError(errNode, wouldWorkWithAwait, leftStr, rightStr)) {
-        errorAndMaybeSuggestAwait(
-          errNode,
-          wouldWorkWithAwait,
-          Diagnostics.Operator_0_cannot_be_applied_to_types_1_and_2,
-          tokenToString(operatorToken.kind),
-          leftStr,
-          rightStr
-        );
-      }
     }
     function tryGiveBetterPrimaryError(errNode, maybeMissingAwait, leftStr, rightStr) {
       switch (operatorToken.kind) {
diff --git a/lib/typescript.js b/lib/typescript.js
index 2643aa12aa6497ca0b90ecd99b202f37ea0e6330..7cf10bb62ba73a3dbd5a4009b07a0ba5641d0516 100644
--- a/lib/typescript.js
+++ b/lib/typescript.js
@@ -84849,16 +84849,6 @@ function createTypeChecker(host) {
         [effectiveLeft, effectiveRight] = getBaseTypesIfUnrelated(leftType, rightType, isRelated);
       }
       const [leftStr, rightStr] = getTypeNamesForErrorDisplay(effectiveLeft, effectiveRight);
-      if (!tryGiveBetterPrimaryError(errNode, wouldWorkWithAwait, leftStr, rightStr)) {
-        errorAndMaybeSuggestAwait(
-          errNode,
-          wouldWorkWithAwait,
-          Diagnostics.Operator_0_cannot_be_applied_to_types_1_and_2,
-          tokenToString(operatorToken.kind),
-          leftStr,
-          rightStr
-        );
-      }
     }
     function tryGiveBetterPrimaryError(errNode, maybeMissingAwait, leftStr, rightStr) {
       switch (operatorToken.kind) {
